src/test/java/BatchCollectTest.java

```diff
@@ -88,8 +88,9 @@ public class BatchCollectTest {
             }
     );
 
-    //@Test
-    public void testCollect() throws StreamingQueryException, InterruptedException {
+    @Test
+    @Disabled
```

Comment:
Why is this test disabled? Add a reason like `@Disabled(value="Broken component xyz")`

Reply:
Looked into it, it was testing the `getCollected()` method, but failing unexpectedly because of how the test data was created in that test. However there was a "TODO: remove" on the function so I just got rid of the method and the test as well. The method wasn't used anywhere in PTH-10 so it was safe to do so. Fixed some other stuff in tests at the same time and added more assertions.